Orange/widgets/visualize/tests/test_owscatterplot.py

```diff
@@ -102,3 +103,26 @@ def test_report_on_empty(self):
         self.widget.report_plot.assert_not_called()
         self.widget.report_caption.assert_not_called()
         self.widget.report_items.assert_not_called()
+
+    def test_data_column_nans(self):
+        """
+        ValueError cannot convert float NaN to integer.
+        In case when all column values are NaN then it throws that error.
+        GH-2061
+        """
+        dat = """\
+            \t\t
+            42.11\t\tneumann
+            17.01\t\tneumann
+            19.01\t\tneumann
+            33.21\t\tturing
+            34.01\t\tturing"""
+        file = io.StringIO(dat)
+        data = TabReader(file).read()
+        self.send_signal("Data", data)
+
+        self.widget.graph.attr_color = self.widget.get_shown_attributes()[1]
```

Comment:
Could you select attributes using helpers found in https://github.com/biolab/orange3/blob/master/Orange/widgets/tests/utils.py#L268?